src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_commands.py

```diff
@@ -37,6 +39,25 @@
 # from .validators import db_up_namespace_processor
 
 
+register_logic_breaking_change('postgres flexible-server create', 'Update default value of "--sku-name"',
+                               target_version='May 2025',
```

Comment:
We use `2.73.0` for next breaking change target version

Reply:
Updated: 
<img width="411" alt="image" src="https://github.com/user-attachments/assets/91db5852-4746-4793-8a09-0201fda0a4e0" />